Add Python Airflow API token instructions (#2092)
astronomer/docs: astro/airflow-api.mdx

### Retrieve an access token

To retrieve an Astro access token, run the following API request with your Deployment API key ID and secret:
<Tabs>

> <TabItem value="sh" label="Shell">
> Using [cURL](https://curl.se/):

Reviewer: ```suggestion

<TabItem value="sh" label="Shell">

Using [cURL](https://curl.se/):

```
I get good luck with builds when I add space between all different kinds of elements.

### Retrieve an access token

> To retrieve an Astro access token, run the following API request with your Deployment API key ID and secret:
> <Tabs groupId="retrieve-an-access-token">
> 
> <TabItem value="sh" label="Shell">
> 
> Using [cURL](https://curl.se/):

Reviewer: ```suggestion
<Tabs groupId="retrieve-an-access-token">

<TabItem value="sh" label="Shell">

To retrieve an Astro access token, run the following API request with your Deployment API key ID and secret using [cURL](https://curl.se/):

```

```sh
curl --location --request POST "https://auth.astronomer.io/oauth/token" \
        --header "content-type: application/json" \
        --data-raw '{
            "client_id": "[api-key-id]",
            "client_secret": "[api-key-secret]",
            "audience": "astronomer-ee",
            "grant_type": "client_credentials"}'
```
</TabItem>
<TabItem value="python" label="Python">

> Using the [`requests`](https://docs.python-requests.org/en/latest/index.html) library:

Reviewer: ```suggestion

To retrieve an Astro access token, use the [`requests`](https://docs.python-requests.org/en/latest/index.html) library to make your API request. For example, your code might look like the following:
```

> ```

Reviewer: ```suggestion

```python
```

def get_api_token() -> str:
  r = requests.post(
      "https://auth.astronomer.io/oauth/token",
      json={
          "client_id": "[api-key-id]",
          "client_secret": "[api-key-secret]",
          "audience": "astronomer-ee",
          "grant_type": "client_credentials"
      }
  )
  r.raise_for_status()
  return r.json()
```

> </TabItem>
> </Tabs>
>   

Reviewer: ```suggestion

</TabItem>
</Tabs>
```
